.github/workflows/start-new-release.yml

```diff
@@ -3,10 +3,6 @@ name: "Start new release"
 on:
   workflow_dispatch:
     inputs:
-      branch:
-        description: 'Start release from this branch'
-        default: 'develop'
-        required: true
```

Comment:
`develop` is the default branch out of the box, so there is no need for the additional input field.